.luacheckrc

```diff
@@ -52,3 +52,23 @@ files["spec/**/*.lua"] = {
 files["spec-old-api/**/*.lua"] = {
     std = "ngx_lua+busted",
 }
+
+files["kong/tools/sandbox.lua"] = {
+    read_globals = {
+        "table.pack",
+        "table.unpack",
+        string = { fields = {rep = { read_only = false } } },
+    }
+}
+
+files["kong/tools/sandbox_helpers.lua"] = {
+    read_globals = {
+        "table.unpack",
+    }
+}
+
+files["spec/01-unit/020-sandbox/02-sandbox_helpers_spec.lua"] = {
+    read_globals = {
+        "table.unpack",
+    }
+}
```

Comment:
any reason to modify the global file instead of using inline `-- luacheck: ignore` markers?

Reply:
if luacheck's `ngx_lua` target doesn't include luajit's 5.2/.3 backports, the more "global" the fix, the better.  I'd suggest adding to the target, not to each file's exception list.